protocol/x/clob/keeper/liquidations.go

```diff
@@ -452,6 +452,35 @@ func (k Keeper) GetFillablePrice(
 		return nil, err
 	}
 
+	// stat liquidation order for negative TNC
+	// CLOB(906) Prevent duplicated stat emissions for liquidation orders in PrepareCheckState.
```

Comment:
Small nit (makes it easier to lookup all TODOs and / or CLOB-906 in the codebase):

```suggestion
	// TODO(CLOB-906): Prevent duplicated stat emissions for liquidation orders in PrepareCheckState.
```